receiver/kafkareceiver/factory.go

```diff
@@ -17,12 +17,14 @@ import (
 )
 
 const (
-	defaultTopic         = "otlp_spans"
-	defaultEncoding      = "otlp_proto"
-	defaultBroker        = "localhost:9092"
-	defaultClientID      = "otel-collector"
-	defaultGroupID       = defaultClientID
-	defaultInitialOffset = offsetLatest
+	defaultTopic             = "otlp_spans"
+	defaultEncoding          = "otlp_proto"
+	defaultBroker            = "localhost:9092"
+	defaultClientID          = "otel-collector"
+	defaultGroupID           = defaultClientID
+	defaultInitialOffset     = offsetLatest
+	defaultSessionTimeout    = 45 * time.Second
```

Comment:
This value and the README don't match. Please update for consistency. (Not sure which is intended to be the official default)

Reply:
(I see now in the original issue description the recommendation was to be at least 30 seconds)

Reply:
good catch, I ran into two different default values between kafka and samara and probably left it in a middle state

Reply:
Samara defaults to 10s so I've defaulted it here to 10s to match the existing behavior.

I'm not sure if it makes sense to set the defaults for the Kafka receiver to the event hub recommendation.